content/en/tracing/service_catalog/guides/integrating-with-ci-visibility.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Integrating with CI Visibility
+kind: guide
+further_reading:
+- link: "/tracing/service_catalog/"
+  tag: "Documentation"
+  text: "Datadog Service Catalog"
+---
+
+In the Delivery lens, you can see CI pipeline and static analysis results associated to your services.
+
+By default, your service is associated to [CI pipeline(s)][4] via repository URL. 
+You can edit the pipelines associated with each service by modifying the ci-pipeline-fingerprints field in your [service metadata][5].
+
+To add or remove a pipeline associated to your service, first click on Edit Metadata in your service page and go to Software Delivery.
```

Comment:
It's either _Edit Metadata_ or `Edit Metadata` ... maybe @alai97 can confirm which one it should be?

Reply:
let's add another sentence after:

Please note that this is only available for [Service Catalog schema v2.2](https://docs.datadoghq.com/tracing/service_catalog/service_definition_api/#further-reading)

Reply:
Resolved except for 

> It's either Edit Metadata or Edit Metadata ... maybe @alai97 can confirm which one it should be?